apps/chat/src/store/import-export/importExport.epics.ts

```diff
@@ -241,27 +248,29 @@ const importConversationsEpic: AppEpic = (action$) =>
       return from(
         ConversationService.setConversations(preparedConversations),
       ).pipe(
+        toArray(),
         switchMap(() => {
           return ConversationService.getConversations(undefined, true).pipe(
-            catchError((err) => {
-              console.error(
-                'An error occurred while uploading conversations and folders:',
-                err,
+            catchError(() => {
+              toast.error(
```

Comment:
use action for showing error instead

Reply:
Thank you for the suggestion!
Changed to use 'UIActions.showErrorToast'